src/wxflow/file_utils.py

```diff
@@ -110,3 +112,27 @@ def _make_dirs(dirlist):
             except Exception as ee:
                 logger.exception(f"Error creating directory {dd}")
                 raise ee
+
+    @staticmethod
+    def _link_files(filelist):
+        """Function to link all files specified in the list
+
+        `filelist` should be in the form:
+        - [link name, target]
+
+        Parameters
+        ----------
+        filelist : list
+                List of lists of [link name, target]
```

Comment:
Does target here mean the first half of `ln` as in `ln target dest`? I think that could be confusing since, to me, the target is where the link is pointing to.

Reply:
Yeah so looking at the pathlib documentation, I think we should be a bit more descriptive here, since many people might blindly think the order is:
`[ target, link_name ]` when it is the opposite here because they are used to how `ln` works

Reply:
So, the signature of `ln` is as follows:
```bash
ln TARGET LINK_NAME
```
The signature of `cp` is:
```bash
cp SOURCE DESTINATION
```
Requirement of  `copy` list in `wxflow` is `[SOURCE, DESTINATION]`

I wrote `link` with the intention to use the list provided by `copy`.  This will allow us to write the yamls as if targeted for ops (using copy), but then replacing the action `copy` with` link` for research (if needed).

I could, if the desire is to switch the implementation in `link` to the correct signature, we will need a method to `map` existing `copy` list into the correct `link` list without having to duplicate the section in the yaml.

Both are easy enough to do.

Reply:
As it stands, `copy` cannot be used as is because `copy` allows the `DESTINATION` to end with `/`; meaning, the `DESTINATION` is a directory and the destination filename is the same as `SOURCE` filename.
I will switch the `link` to the correct form and for now let users write the section of `link` yaml as syntax would want it.

Reply:
Thanks for clearing up my confusion.  I will correct the mistake.

Reply:
Thanks @CoryMartin-NOAA. I have applied the suggestions via 2e16ce2